SU-SOE/stanford_soe_helper
# Non-accessible HTML on collection pages

# READY FOR REVIEW

# Summary
This fixes the Non-accessible HTML on collection pages. Changing <div> <span> <div> on the magazine article collection views to <div> <div> <div>.

# Needed By (Date)
- Sprint end

# Criticality
- fixes the Non-accessible HTML on collection pages


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-2884 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2884)
- `drush fr stanford_soe_helper_collection_views`
- `drush cc all`
- go to: `magazine/future-everything` and make sure the span tags are now divs for view results.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2884

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
